changelog.d/359.bugfix

```diff
@@ -0,0 +1 @@
+Get UserAgent from package.json
```

Comment:
```suggestion
Match UserAgent version to Hookshot's version
```

To be a little more precise? As-is you may think that the whole useragent is configurable in package.json :)